src/variables-dark.css

```diff
@@ -20,4 +20,7 @@
   --variable: #d941e2;
   --highlight: #efdb43;
   --select-arrow: svg-load('./assets/select-arrow.svg', fill: #efefef);
+  --meter-optimum:#39a33c;
```

Comment:
Really tiny, but here and in `variables-light.css` there's a space missing, can you add that after the `:`? Color choices are awesome btw.